examples/db_connector/3_ydb.py

```diff
@@ -20,20 +24,22 @@
 # ##### or use local-ydb with variables from .env_file
 
 # db_uri="grpc://localhost:2136/local"
-db_uri = "{}{}".format(
-    os.getenv("YDB_ENDPOINT"),
-    os.getenv("YDB_DATABASE"),
-)
-db = connector_factory(db_uri)
-
-pipeline = Pipeline.from_script(
-    TOY_SCRIPT,
-    context_storage=db,
-    start_label=("greeting_flow", "start_node"),
-    fallback_label=("greeting_flow", "fallback_node"),
-)
-
-if __name__ == "__main__":
-    check_happy_path(pipeline, HAPPY_PATH)
-    if is_interactive_mode():
-        run_interactive_mode(pipeline)
+# db_uri = "{}{}".format(
```

Comment:
Why? If it comment, how we will check that examples will work

Reply:
Uncomment